### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by NomadBear7** [JaehaKim97/Adaptive-Downsampling-Model#4]
Thanks for your great work.I got a problem about training process，this is my down_loss_graph , as you can see ,the loss_g and the loss_dl increases significantly with the epoch, what causes this issue?(I only do the Downsample)

![Image](https://github.com/user-attachments/assets/c50743d5-390f-4aec-abf4-deb14cdba4b2)

**Comment by JaehaKim97** [comment]
Hi @NomadBear7,

Sorry for the late reply.

In my experience, it is quite common for the generator loss (orange) to increase relative to the discriminator loss (blue) along the training.
This is because simulating the distribution of target images is typically much more difficult than distinguishing between generated and real images.
(I've attached a graph of one of my training runs where you can observe this tendency.)

![Image](https://github.com/user-attachments/assets/578d6621-6d27-46c5-94b0-2b1e8508b659)

However, in your case, it seems that the discriminator loss becomes very small, resulting in a very large generator loss.
In my guessing, this can happen if (1) the distribution of source images is significantly different from that of target images, or (2) the number of target images is too small, making the discrimination task too easy.

I recommend reorganizing the source and target datasets with above points in mind.
Please feel free to let me know if you encounter any further issues.

Thank you for your interest in our work!

Best regards,
Jaeha

**Comment by NomadBear7** [comment]
Thank you for your response! I used the dataset you mentioned in Dataset preparation for training. Did you also use the DIV2K and RealSR datasets for training and get the loss graph you show me?

**Comment by JaehaKim97** [comment]
Sorry for the significant delay.
(I actually lost my experimental data and had to re-run everything.)

I observed the following training curve:
![Image](https://github.com/user-attachments/assets/25807a5d-af3d-4618-802b-14363c5e6066)

I suspect that the particularly high loss_g in your case may be due to the training data.
DIV2K has a distribution that differs substantially from RealSR, making it easier for the discriminator to distinguish between real and generated samples.
For the RealSR experiment, we first split the training data and used each half for HR and LR, as described in the manuscript.
This helps to reduce the distribution gap between the HR and LR sets.

Even if loss_g appears high, please note that the simulated LR images are still closer to target degradations than standard bicubic downsampling.
So using them remains meaningful and beneficial.

Sincerely,
Jaeha

**Comment by NomadBear7** [comment]
Thank you very much for your reply. It has been extremely helpful to me. I sincerely express my gratitude to you！
